assets/docker-compose/policies/todoApp.GET.rego

```diff
@@ -0,0 +1,3 @@
+package todoApp.GET.todos
```

Comment:
@apoland I would actually add all the policy files from the `aserto-policies/policy-todo-rebac` repo, since that would line this configuration up with the rest of the getting started materials (which use policy-todo-rebac).

Reply:
Updated with the full policy and sample data